src/org/usfirst/frc/team2706/robot/subsystems/DriveTrain.java

```diff
@@ -26,6 +29,11 @@
 	private Encoder left_encoder, right_encoder;
 	private AnalogInput rangefinder;
 	private AnalogGyro gyro;
```

Comment:
The first choice GYRO is not ANALOG. It is a SPI part.
The chip we're getting is the ADXRS450 and I found a wpilib library for the ADXRS453@ https://gist.github.com/jcorcoran/6d721cf6570f772b51f8

Not sure what the differences are but it can't be that much. They seem to have the same specs, it almost looks like its a different package rather than feature.